boost_test_schedule

```diff
@@ -53,6 +53,7 @@ test_gui2/modal_dialog_test_wml_error
 test_gui2/modal_dialog_test_wml_message_left
 test_gui2/modal_dialog_test_wml_message_right
 test_gui2/modal_dialog_test_wml_message_double
+test_gui2/modal_dialog_test_achievements_dialog
```

Comment:
The lines around this are in alphabetical order, please move this one.

Reply:
Updated.